Platon/backend/tests/profile_management_test.py

```diff
@@ -425,14 +422,29 @@ def test_fetch_GS_info(self):
 class NotificationTests(BaseTest):
 
     def setUp(self):
+        # Add artificial users to test login feature
+        jobs = [
+            Jobs("Academician"),
+            Jobs("PhD Student")
+        ]
+
+        for job in jobs:
+            db.session.add(job)
+
+        db.session.commit()
         # Add artificia users to test login feature
         users = [
-            User("umut@deneme.com",True,"b73ec5e4625ffcb6d0d70826f33be7a75d45b37046e26c4b60d9111266d70e32",3.5,"Umut","Özdemir",False,None,None,None),
-            User("can@deneme.com",False,"cce0c2170d1ae52e099c716165d80119ee36840e3252e57f2b2b4d6bb111d8a5",4.6,"Can","Deneme",True,None,None,None)
+            User("umut@deneme.com", True, "b73ec5e4625ffcb6d0d70826f33be7a75d45b37046e26c4b60d9111266d70e32", 3.5,
+                 "Umut", "Özdemir", False, None, None, None, 1, "boun"),
+            User("can@deneme.com", False, "cce0c2170d1ae52e099c716165d80119ee36840e3252e57f2b2b4d6bb111d8a5", 3.4,
+                 "Can", "Deneme", False, None, None, None, 2, "boun")
         ]
         for user in users:
             db.session.add(user)
         db.session.commit()
+        for user in users:
```

Comment:
users are already added into db, a few line above. this seems duplicate to me. am I mistaken?